.eslintrc.js

```diff
@@ -32,10 +32,11 @@ const config = {
     "prettier/prettier": [
       "error",
       {
-        "endOfLine": "auto"
-      }
-    ]    
-    
+        endOfLine: "auto",
+      },
+    ],
+    "jsdoc/require-description-complete-sentence": "off",
+    "next/no-img-element": "off",
```

Comment:
I think it is because there are some deployment issues each time I committed my code. So I thought there is some bugs I need to fix. I updated it according to the error message of the deployment.